rules/android_library/impl.bzl

```diff
@@ -555,7 +555,6 @@ def finalize(
             ),
             _direct_source_jars = depset([ctx.outputs.lib_src_jar]),
             _hidden_top_level_INTERNAL_ = depset(
-                resources_ctx.validation_results,
```

Comment:
Should these be added to validation_outputs below, or are these already included? I looked through the code and I'm not sure that they're already there, but maybe I missed it